askama_shared/src/parser.rs

```diff
@@ -660,6 +676,10 @@ expr_prec_layer!(expr_compare, expr_bor, "==", "!=", ">=", ">", "<=", "<");
 expr_prec_layer!(expr_and, expr_compare, "&&");
 expr_prec_layer!(expr_or, expr_and, "||");
 
+fn expr_minus_or_plus(i: &str) -> IResult<&str, Sign> {
```

Comment:
This name, too, is a bit too literal for me, something like `expr_handle_ws` maybe?

Reply:
Ok!